Resources/Prototypes/_RMC14/Entities/Clothing/Head/welding.yml

```diff
@@ -0,0 +1,31 @@
+# Welding mask(s)
+- type: entity
+  parent: ClothingHeadBase
+  id: RMCWeldingMaskBase
+  name: welding mask
+  abstract: true
+  components:
+  - type: Item
+    storedRotation: 0
+  - type: IngestionBlocker
+  - type: FlashImmunity
+  - type: IdentityBlocker
+  - type: EyeProtection # This cancels out the upstream RequiresEyeProtection on welders
+  - type: RMCEyeProtectionItem
+    raisedEquippedPrefix: up
+    slots:
+    - head
+  - type: HideLayerClothing
+    slots:
+    - Snout
+
+- type: entity
+  parent: RMCWeldingMaskBase
+  id: RMCWeldingMask
+  name: welding mask
```

Comment:
```suggestion
```

doesnt need the name since the parent already has the same name

Reply:
Sure, that can be removed